store/src/java/com/zimbra/cs/service/FeedManager.java

```diff
@@ -185,6 +193,28 @@ public void cleanup() {
         }
     }
 
+    /**
+     * Returns true if target address is private, link-local, loopback, or individually blacklisted.
+     * @param url The target
+     * @return True if address is blocked for feed manager
+     */
+    protected static boolean isBlockedFeedAddress(HttpURL url) {
+        List<String> blacklist = Arrays.asList(LC.zimbra_feed_manager_ip_blacklist.value().split(","));
```

Comment:
Can we support CIDR notation to allow for easier blocking of a large range of addresses?

Reply:
Done and moved private addresses to default config rather than forced off.